db/migrate/20170227165349_add_image_file_to_facilities.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageFileToFacilities < ActiveRecord::Migration
+  def change
+    add_column :facilities, :image_file, :string
```

Comment:
The thing is that image is the name of the icon and it's a text icon sounds like a different version for the image. When you say image_file it does say it's a file, not a text.